README.md

```diff
@@ -31,7 +31,7 @@ Please go through the [Contribution Guidelines](CONTRIBUTING.md) before going fo
 **Note:** You might want to use the docker instructions if you do not want to setup your own environment.
 
 * Install ruby using RVM, use ruby-2.6.5
-* Install Dependencies: `bundle install `
+* Install Dependencies: `bundle install ` (Note: If you do not have bundler already installed , install it using `gem install bundler`)
```

Comment:
Since this step just after installing ruby, it is better assume that bundler is not installed and add this as a separate step above

Reply:
K